File: opcua-webservice-spring/core/src/main/java/org/opcfoundation/webserver/types/message/digitaltwin/GetElementsResponse.java
package org.opcfoundation.webserver.types.message.digitaltwin;

import java.util.HashSet;
import java.util.Set;

public class GetElementsResponse {
    private final Set<String> elementNames;

    public GetElementsResponse()
    {
        this.elementNames = new HashSet<>();
    }

    public void add(String elementName)
    {
        elementNames.add(elementName);
    }

    public Set<String> getElementNames() {
        return elementNames;
    }
}

File: opcua-webservice-node/src/server/service/message/digitialtwin/GetElementsResponse.ts
export class GetElementsResponse {
    private readonly _elementNames: Set<string>;

    constructor() {
        this._elementNames = new Set();
    }

    add(elementName: string): void {
        this._elementNames.add(elementName);
    }

    get elementNames(): Set<string> {
        return this._elementNames;
    }
}